### PR TITLE
Move modal styling into webpack

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Ports over styling around modals out of sprockets and into webpack.

This pull request makes the following changes:
* port over modal styling overrides into their own webpack module
* delete some unused or ineffective styling that doesn't actually take effect anymore (see comments) 

no view changes

It relates to the following issue #s: 
* Bumps #1854 

For reviewer:
* Adjust the title to explain what it does for the notification email to the listserv.
* Tag this PR:
  * `feature` if it contains a feature, fix, or similar. This is anything that contains a user-facing fix in some way, such as frontend changes, alterations to backend behavior, or bug fixes.
  * `dependencies` if it contains library upgrades or similar. This is anything that upgrades any dependency, such as a Gemfile update or npm package upgrade.
* If it contains neither, no need to tag this PR.

### CODE REVIEW

File: app/assets/stylesheets/patients.scss [hunk 1]
```diff
@@ -10,14 +10,6 @@
     cursor: pointer;
 }
 
-.partial_fulfillment_modal {
-  padding: 5%;
-}
-
-.pledge_modal_category {
```
Comment (colinxfleming) [inline app/assets/stylesheets/patients.scss:17]: not used anywhere! removed it

**Review by colinxfleming — COMMENTED** [review]
Sprockets bootstrap and npm bootstrap are clashing a little, so I'm cheating a little and marking some stuff `!important` to hack around the problem for right now.

**Review by lomky — APPROVED** [review]
🎉

**Comment by colinxfleming** [comment]
thank you @lomky !

**Pull request merged**